README.md

```diff
@@ -133,7 +184,21 @@ app.on("delete", "/users/([^/]*)", function(context) {
 
 <div id="rocky_use"><h3>use(<i>callback</i>)</h3></div>
```

Comment:
Parameter for this function can be a single middleware function or an array of middleware functions.